themes/digital.gov/layouts/resources/single.html

```diff
@@ -8,7 +8,7 @@
         <div class="grid-row">
           <div class="grid-col-12" data-edit-this="{{- $path -}}">
             <p class="breadcrumb">
-              <a href="{{- "resources/" | absURL -}}">
+              <a href="{{- "resources/" | absURL -}}" aria-label="Back to Resources">
```

Comment:
Should we clarify that it's a link back to the landing pages for resources?

For example _Back to Resources landing page_

Reply:
Updated